recipes/moka-py/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "moka-py" %}
+{% set version = "0.1.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/moka_py-{{ version }}.tar.gz
+  sha256: 33065280a8cad9298b62bc928c6a4e0cf61df82fa60cac95d4d62c095e0c216c
+
+build:
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
```

Comment:
Does this one has any 3rd party license that needs to be packaged?

Reply:
Oops I forgot about that part my bad, let me add the licenses